scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -969,8 +890,6 @@ mod tests {
             "DELETE FROM RANGE_DELETE WHERE pk1 = 0 AND pk2 = 0 AND (ck1, ck2, ck3) > (3, 3, 3)",
         ]);
 
-        test_replication(&get_uri(), schema, operations)
-            .await
-            .unwrap();
+        test_replication(schema, operations).await.unwrap();
```

Comment:
The last-but-one commit in the PR does not compile. Probably because this change belongs to the last-but-one commit, not the last one.

Reply:
This one is a rebasing hell at this point 😩